Исправил вывод в I2CScanner.h

irongamer54 opened the pull request.

irongamer54 committed:
Update I2CScanner.h

TeaCupMe commented on src/I2CScanner.h:
```diff
@@ -66,23 +66,23 @@ class I2CScanner final {
 		ISL_StatusTypeDef status = i2c.isReady(addr);
 		if (status != ISL_OK) {
-			printTextToUART(uart, "Device if not available!\n");
+			printTextToUART(uart, "Device if not available!\n\r");
```
Device ->**is**<- not available

TeaCupMe commented on src/I2CScanner.h:
```diff
@@ -92,20 +92,20 @@ class I2CScanner final {
 			status = i2c.readMem(addr, firstRegister, val, nRegisters);
 			if (status != ISL_OK) {
-				printTextToUART(uart, "Cant read device registers!\n");
+				printTextToUART(uart, "Cant read device registers!\n\r");
```
**can't**

TeaCupMe commented:
И пробегись ещё раз по всему файлу, проверь визуально, что всё остальное нормально. Понимаю, что изначально не файл писал не ты, но если уж ты его правишь, лучше сразу привести к окончательно нормальному виду

irongamer54 committed:
Изменение вывода I2C сканера

Вывод битовых данных был центрирован, в сканирование регистров пачкой был добавлен заголовок таблицы

irongamer54 committed:
Добавлен пример полного использования библиотеке

irongamer54 commented:
Добавил пример, а также сделал вывод чуть более читаемым

TeaCupMe reviewed (approved):
lgtm
